generator/transit_generator.cpp

```diff
@@ -319,7 +256,182 @@ void BuildTransit(string const & mwmDir, string const & countryId,
   w.Seek(startOffset);
   header.Visit(serializer);
   w.Seek(endOffset);
+
   LOG(LINFO, (TRANSIT_FILE_TAG, "section is ready. Header:", header));
 }
+
+void GraphData::Append(GraphData const & rhs)
+{
+  ::Append(rhs.m_stops, m_stops);
+  ::Append(rhs.m_gates, m_gates);
+  ::Append(rhs.m_edges, m_edges);
+  ::Append(rhs.m_transfers, m_transfers);
+  ::Append(rhs.m_lines, m_lines);
+  ::Append(rhs.m_shapes, m_shapes);
+  ::Append(rhs.m_networks, m_networks);
+}
+
+void GraphData::Clear()
+{
+  ClearVisitor const v;
```

Comment:
```ClearVisitor{}(v);``` ?

Reply:
ClearVisitor doesn't have operator()(GraphData &). It only can call  method clear() for fields.

Reply:
OK